Update catchability section with env Q

e-perl-NOAA opened the pull request:
See ss3-source-code PR [#596](https://github.com/nmfs-ost/ss3-source-code/pull/596) and issue [#595](https://github.com/nmfs-ost/ss3-source-code/issues/595).

@Rick-Methot-NOAA Can you make sure the extra parameters table example looks okay?
@iantaylor-NOAA Could you take a look just to make sure it all makes sense?

Rick-Methot-NOAA reviewed (approved):
Good news.  I found the artifact all by myself.
One small change:
8.8.1 Mirrored Q with scale
better as:
8.8.1  Example parameter lines

e-perl-NOAA commented:
@iantaylor-NOAA could you take a quick look at this just to make sure everything makes sense?

iantaylor-NOAA reviewed (approved):
Looks great. Thank you @e-perl-NOAA and @Rick-Methot-NOAA for cleaning up this section and documenting the new options. Sorry I was out sick so this response is slow.

I'm not sure why the github action to link the artifacts is being skipped, but tracking them down the old way shows me that there's a quirk in how the numbered lists work in HTML vs PDF. In the HTML case, it's a little strange to have the repeated numbers (as opposed to letters preceding the numbers in PDF). See screenshot below.

I also see a misspelled "requrired".

![image](https://github.com/nmfs-ost/ss3-doc/assets/4992918/b2b86b4a-5f51-4385-8f02-689c6a894824)

Rick-Methot-NOAA commented:
Looks good regarding my requested change.
Acknowledging Ian's additional comments.

e-perl-NOAA commented:
@iantaylor-NOAA they should be lettered now. I added another condition to the github action that adds the artifacts because I think it was only adding them on the initial PR, not on subsequent pushes but hopefully what I added will get it adding the artifacts on pushes.
